src/main/resources/swagger/api-docs.yaml

```diff
@@ -435,6 +435,43 @@ paths:
           description: Error parsing source method configuration
         500:
           description: Internal Server Error
+  /workspaces/{workspaceNamespace}/{workspaceName}/method_configs/copyToMethodRepo:
+    post:
+      tags:
+        - Method Configurations
+        - Method Repository
+      operationId: copyToMethodRepo
+      summary: Copy a Method Config in a workspace to the Method Repository
+      produces: application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: configToCopy
+          description: Method Configuration to Copy
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: PublishConfigurationIngest
+      responses:
+        201:
```

Comment:
these response codes don't match what's documented for rawls at https://swagger.dsde-dev.broadinstitute.org/#!/methodconfigs/copy_configuration_to_repo. I assume this is a problem with the rawls swagger?